Yoast/yoast-components
# Add focus style to wizard navigation buttons.

## Summary

This PR can be summarized in the following changelog entry:

* Adds focus style to the Configuration Wizard navigation buttons

## Relevant technical choices:

- adds a basic focus style
- adds a new `#5b9dd9` blue color to the palette: this is the color used by WordPress for focus

## Test instructions

- go in the configuration wizard in the standalone app
- tab with the keyboard to the "previous" and "next" buttons
- see there's a focus style

<img width="1245" alt="screenshot 2019-02-27 at 14 39 30" src="https://user-images.githubusercontent.com/1682452/53495905-3384d600-3aa1-11e9-9687-732d13e47e58.png">

<img width="598" alt="screenshot 2019-02-27 at 14 53 08" src="https://user-images.githubusercontent.com/1682452/53495906-3384d600-3aa1-11e9-963d-0d82232f817e.png">

Note:
while working on this I've noticed the state of the CSS could be improved. There are duplications of rules between the plugin and the yoast-components. The preview of the Wizard in the standalone app looks different from the Wizard in the plugin. This doesn't help in testing. Also, there shouldn't be hardcoded colors in the yoast-components (only colors from the palette should be used).
Will create a separate issue.


## UI changes

Fixes #833
